src/main/java/picard/util/IntervalListScatterer.java

```diff
@@ -16,30 +15,30 @@
 
     public enum Mode {
         /**
-         * A simple scatter approach in which all output intervals have size equal to the total base count of the source list divide by the
-         * scatter count (except for possible variance in the final interval list).
+         * A simple scatter approach in which all output intervals have size equal to the total base count of the source list divided by the
```

Comment:
BALANCING_WITHOUT_INTERVAL_SUBDIVISION_WITH_OVERFLOW does not need a numbered list element as there is only one item on the list.

Looks like:

![screenshot 2017-12-11 20 17 50](https://user-images.githubusercontent.com/11543866/33862372-6325aef0-deb0-11e7-8647-de87b3c7afe3.png)